packages/bun-vscode/package.json

```diff
@@ -328,5 +357,8 @@
         }
       }
     ]
+  },
+  "dependencies": {
+    "strip-ansi": "^7.1.0"
```

Comment:
Let's remove this dependency, we don't need it. Can be 1 line utility somewhere

Reply:
Sure, removed.